docs/content/en/docs/install/install.md

```diff
@@ -76,67 +88,104 @@ Some helpful hints:
+| [cert-manager](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/README.md)            | [TLS Certificate management for all Keptn components](../architecture/cert-manager)                                                                       | [keptn-cert-manager/values.yaml](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/values.yaml) |
+
 ## Customizing the configuration of components
 
 To modify configuration of the subcomponents of Keptn,
-use the corresponding Helm values in your main `values.yaml`.
+use the corresponding Helm values in your `keptn/values.yaml` file.
 Use the subcomponent's parent value as the root for your configuration.
 
 Here is an example `values.yaml` altering global and metrics operator values:
 
 {{< docsembed path="content/en/docs/install/assets/values-advance-changes.yaml" >}}
 
-### Modify Helm configuration options
+Note the additional values that are specified
+in the `metricsOperator` section.
+These are documented in the README file for that operator,
+which is linked from the `metrics-operator` item under "Component"
+in the table above.
+To implement this:
+
+* Go into the `values.yaml` file linked under "Configuration file"
+* Copy the lines for the values you want to modify
+* Paste those lines into the `keptn/values.yaml` file
+  and modify their values in that file
 
-Helm values can be modified before the installation.
+### Modify Helm configuration options
 
-To modify configuration options, download a copy of the
-default `values.yaml` file,
-modify some values, and use the modified file to install Keptn:
+To modify Helm configuration options:
 
-1. Download the `values.yaml` file:
+1. Download a copy of the `keptn/values.yaml` file:
 
    ```shell
    helm show values keptn/keptn > values.yaml
    ```
 
 1. Edit your local copy to modify some values
 
-1. Install Keptn by adding the following string to your `helm upgrade` command line:
+1. Add the following string
+   to your `helm upgrade` command line to install Keptn
+   with your configuration changes:
 
    ```shell
    --values=values.yaml
    ```
 
-You can also use the `--set` flag
-to specify a value change for the `helm upgrade --install` command.
-Configuration options are specified using the format:
-
-```shell
---set key1=value1 \
---set key2=value2 ...
-```
-
-The [helm-chart](https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/README.md) page
-contains the full list of available values.
+   For example, if you create a `my.values.yaml`
+   and modified some configuration values,
```

Comment:
the tenses are weird here... not sure how to fix it the proper way though 😅

Reply:
You are right -- I made them both present tense.